### PR TITLE
Packagist SearchCommand Added

### PR DESCRIPTION
**Pull request by Ninjavin**
Hey, @pedrofracassi I have added the Packagist SearchCommand. Please review this PR and tell me about any subsequent changes.

### CODE REVIEW

File: src/commands/utility/packagist.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      requirements: { apis: ['packagist'] },
+      embedColor: Constants.PACKAGIST_COLOR,
+      embedLogoURL: 'https://i.imgur.com/MLTCgKN.png'
+    }, client)
+  }
+
+  async search (_, query) {
+    const { data } = await this.client.apis.packagist.search(query)
+    var queryArray = Object.keys(data.results).map(i => data.results[i])
+    return queryArray
```
Comment (metehus) [inline src/commands/utility/packagist.js:24]:
```suggestion
    return Object.keys(data.results).map(i => data.results[i])
```

**Review by metehus — DISMISSED** [review]

File: src/commands/utility/packagist.js (hunk 1 above)
Comment (Ninjavin) [reply src/commands/utility/packagist.js]: @metehus Requested Changes have been made. You may review it. Thanks

File: src/commands/utility/packagist.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      }],
+      requirements: { apis: ['packagist'] },
+      embedColor: Constants.PACKAGIST_COLOR,
+      embedLogoURL: 'https://i.imgur.com/MLTCgKN.png'
+    }, client)
+  }
+
+  async search (_, query) {
+    const { data } = await this.client.apis.packagist.search(query)
+    return Object.keys(data.results).map(i => data.results[i])
```
Comment (almeidx) [inline src/commands/utility/packagist.js:23]: ?
```suggestion
    return Object.values(data.results)
```

File: src/commands/utility/packagist.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  async handleResult ({ channel, author, language }, item) {
+    const embed = new SwitchbladeEmbed(author)
+    moment.locale(language)
+    channel.startTyping()
+    embed
+      .setColor(Constants.PACKAGIST_COLOR)
+      .setAuthor('Packagist', this.embedLogoURL, 'https://packagist.org/')
+      .setDescriptionFromBlockArray([
+        [
+          `[${item.name}](${item.url})`, item.description ? item.description : null
```
Comment (almeidx) [inline src/commands/utility/packagist.js:39]:
```suggestion
          `[${item.name}](${item.url})`,
          item.description ? item.description : null
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: src/commands/utility/packagist.js (hunk 3 above)
Comment (Ninjavin) [reply src/commands/utility/packagist.js]: Fixed!

File: src/commands/utility/packagist.js (hunk 2 above)
Comment (Ninjavin) [reply src/commands/utility/packagist.js]: Fixed!

**Review by almeidx — APPROVED** [review]
LGTM

**Review by metehus — APPROVED** [review]
LGTM

**Pull request merged**